test/e2e/dns_autoscaling.go

```diff
@@ -293,3 +342,34 @@ func waitForDNSConfigMapCreated(c clientset.Interface, timeout time.Duration) (c
 	}
 	return configMap, nil
 }
+
+func waitForDNSPodsRunningWithNum(c clientset.Interface, expNum int, timeout time.Duration) error {
+	var current int
+	condition := func() (bool, error) {
+		label := labels.SelectorFromSet(labels.Set(map[string]string{ClusterAddonLabelKey: DNSLabelName}))
+		listOpts := v1.ListOptions{LabelSelector: label.String()}
+		pods, err := c.Core().Pods(DNSNamespace).List(listOpts)
```

Comment:
I think there's already a const for kube-system namespace, api.NamespaceSystem maybe? search around in the e2e folder